docs/Remote Control Guide/index.md

```diff
@@ -0,0 +1,184 @@
+
+[View **GetInteriorVehicleData** in the HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/getinteriorvehicledata)
+
+### OnInteriorVehicleData
+
+OnInteriorVehicleData is a notification sent out by SDL when an update is made to a remote control module. You can subscribe to these notifications via GetInteriorVehicleData. This RPC will come with a `ModuleData` structure identifying the changed module and containing the control data object with the new state.
+
+[View **OnInteriorVehicleData** in the HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/oninteriorvehicledata)
+
+### SetInteriorVehicleData
+
+SetInteriorVehicleData is used to set the values of a remote control module by passing in a `ModuleData` structure. The `moduleType` and `moduleId` fields are used to identify the targeted module, and the changes in the respective control data object are applied to that module.
+
+[View **SetInteriorVehicleData** in the HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/setinteriorvehicledata)
+
+### OnRemoteControlSettings
+
+OnRemoteControlSettings is used to notify SDL when passengers of a vehicle change whether remote control is allowed or not via the HMI or if they change the access mode that will be used for resource allocation. It contains a self-explainatory boolean parameter `allowed` and can also contain an access mode, one of auto allow, auto deny, or ask driver.
+
+[View **OnRemoteControlSettings** in the HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/onremotecontrolsettings)
+
+### OnRCStatus
+
+OnRCStatus is a notification sent out by SDL when an update is made to a remote control module's availability. When SDL either allocates a module to an app, or deallocates it from an app, SDL will send OnRCStatus to both the application and the HMI. This notification contains two lists, one describing the modules that are allocated to the application and the other describing the free modules that can be accessed by the application. This notification also contains an `allowed` parameter, which indicates to apps whether or not remote control is currently allowed. If `allowed` is false, both module lists will be empty.
+[View **OnRCStatus** in the HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/onrcstatus)
+
+### GetInteriorVehicleDataConsent
+
+GetInteriorVehicleDataConsent is a request used to reserve a remote control module. If a module does not allow multiple access, only the application that requested consent first (excluding takeover situations described in the Consent section) will be able to interact with that module. This request requires a `moduleType` and `moduleId` to identify the target module.
```

Comment:
plurality added in https://github.com/smartdevicelink/sdl_core_guides/pull/61/commits/1ef97a863cfbfb865722057695a75b2e7f8f2407

Reply:
Regarding 
```
If a module does not allow multiple access, only the application that requested 
consent first (excluding takeover situations described in the Consent section) 
will be able to interact with that module
```
I don't think `(excluding takeover situations described in the Consent section)` is true here since the takeover situations(rules for AUTO_ALLOW, ASK_DRIVER) only apply when `allowMultipleAccess=true`.
```
If a module does not allow multiple access, only the application that requested consent first will be able to interact with that module. Otherwise, if the module does allow multiple access, the rules specified in the (consent)[<link_here>] section apply
````
might work better.

Reply:
good catch! fixed in https://github.com/smartdevicelink/sdl_core_guides/pull/61/commits/4bfc036695cde776df4824670769a2273ec886b8